clients/java/signalr/build.gradle

```diff
@@ -20,6 +21,22 @@ dependencies {
     implementation 'com.google.code.gson:gson:2.8.5'
 }
 
+spotless {
+    java {
+        licenseHeader '// Copyright (c) .NET Foundation. All rights reserved.\n' +
+        '// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.\n' // License header
+
+        importOrder 'java', 'javax', 'org', 'com', 'com.diffplug', ''	// A sequence of package names
+
+        replace      'Not enough space after if', 'if(', 'if ('
+        replace      'Not enough space after )', '){', ') {'
+        replaceRegex 'Too much space after if', 'if +\\(', 'if ('
+        trimTrailingWhitespace()
```

Comment:
Couple rules that would be nice but might be difficult to do:
* no spaces before a comma (not 100% sure about this one)
* single space around all the comparison operators (sounds hard)

Reply:
It's disappointing that Spotless doesn't have a way to configure specific rulesets... I'd hoped we could still use standard rules and just turn some of them off.